web-app-router/components/Input/index.tsx

```diff
@@ -1,7 +1,99 @@
-export const Input = () => {
+"use client";
+import clsx from "clsx";
+import { FloatingLabel } from "flowbite-react";
+import { ComponentProps, memo } from "react";
+import { Icon } from "../Icon";
+
+type InputProps = ComponentProps<"input"> & {
```

Comment:
@andy-t-wang We need to add `register: RegisterReturn` (don't remember exact type name)  prop to make compatible with `react-hook-form`. Also it would be great to have `errors` prop to pass field error object from react-hook-form.